playbooks/roles/prereqs/tasks/setup_nested_virt.yml

```diff
@@ -1,45 +1,42 @@
 ---
-# Check and enable virtualization if necessary
-#  REBOOT WILL BE REQUIRED IF ENABLED
-- name: Check virtualization settings
+- name: "Check virtualization settings"
   shell: cat /proc/cpuinfo | grep vmx
   register: cpuinfo
   ignore_errors: true
+  when: skip_prereqs == false
 
-- name: Output of cpuinfo from proc
+- name: "Output of cpuinfo from proc"
   debug:
     msg: "cpuinfo: {{ cpuinfo.stdout }}"
 
-- name: Exit if virtualization is not turned on in the BIOS
+- name: "Exit if virtualization is not turned on in the BIOS"
   fail: msg="Please enable Intel virtualization (VT) within the host machine's BIOS"
   when: "cpuinfo.stdout == ''"
 
-- name: Check virtualization settings
-  shell: cat /sys/module/kvm_intel/parameters/nested
+- name: "Check virtualization settings"
+  command: cat /sys/module/kvm_intel/parameters/nested
   register: nested
+  when: skip_prereqs == false
```

Comment:
I think this still needs to be done and stop if not set appropriately

Reply:
Into playbooks/ directory:
using: $ ansible-lint setup.yml 

ANSIBLE0012] Commands should not change things if nothing needs doing
/home/firemanxbr/GitHub/contra-env-setup/playbooks/roles/prereqs/tasks/setup_nested_virt.yml:16
Task/Handler: Check virtualization settings

In this case all tasks should have a 'when' to determine when needs doing. Looking the logic if I don't set to skip all 'prereqs' this will works.